Documentation/Input/Pointers.md

```diff
@@ -72,7 +72,6 @@ The *TouchPointer* is responsible for working with Unity Touch input (i.e. touch
 The *MousePointer* powers a screen to world raycast for far interactions, but for mouse instead of touch.
 
 <img src="../../Documentation/Images/Pointers/MRTK_MousePointer.png" width="400">
-<br/>
```

Comment:
I think without this it actually looks weird

Reply:
Do you have a screenshot? I don't see much of a difference on my end

Reply:
I recall seeing after publishing once I could view it on https://microsoft.github.io/

I'm not sure how you can view a private branch on https://microsoft.github.io/?

Reply:
What I see, in the MS Docs renderer for VS Code (which I think should mostly match our github.io styling?):

As-is:
![image](https://user-images.githubusercontent.com/3580640/70748667-476f9080-1cdf-11ea-9472-f7af3c2247b7.png)

With this change:
![image](https://user-images.githubusercontent.com/3580640/70748657-40e11900-1cdf-11ea-87e4-690249aa44f8.png)

Reply:
>I recall seeing after publishing once I could view it

Ahh okay. I'll build DocFx locally and check

Reply:
Yea I've learned not to trust vs code unfortunately

Reply:
Do you have https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-preview?

Reply:
Ah you're right it looks different in the generated doc (more compressed), but I feel like it looks the same on https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/Pointers.html#far-pointers?

Reply:
![image](https://user-images.githubusercontent.com/3580640/70749105-54d94a80-1ce0-11ea-848c-6c6ecc8253b7.png)

Reply:
hmm try switching html img tag to markdown

Reply:
I replaced the img tag with markdown and resized the image to the 400 pixels specified in the img tag